Sources/Apodini/Configurations/VariableWidthIntegerConfiguration.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  Created by Nityananda on 28.01.21.
+//
+
+public struct VariableWidthIntegerConfiguration: Configuration {
+    // MARK: Nested Types
+    public enum Option: Int {
```

Comment:
nit: Maybe a name more precise than `Option` would be better? Something like `IntWidthOption` 🤔